tests_smoke/smoke_test.py

```diff
@@ -5,7 +5,7 @@
 from smoke.test_api_one_off import test_api_one_off
 
 if __name__ == "__main__":
-    for notification_type in [Notification_type.EMAIL, Notification_type.SMS]:
```

Comment:
just for now, we're approaching our SMS spending limit. We've put in for an increase and will change this back after it comes in.